arches/app/utils/index_database.py

```diff
@@ -327,30 +327,41 @@ def index_resources_by_type(
 
     for resource_type in resource_types:
         start = datetime.now()
+        try:
+            uuid.UUID(str(resource_type))
+            graph = models.GraphModel.objects.get(graphid=str(resource_type))
```

Comment:
This part will have to change because slugs are no longer unique in v8 -- they are shared across graph versions.

Reply:
I don't have a functioning v8 setup here so I'll just put this PR on hold for now until I have the time to get one up and running to sort this out.